src/Home.js

```diff
@@ -1,22 +1,85 @@
-import React, { useState, useEffect } from 'react';
-import { StyleSheet, Text, View, FlatList, Pressable, Modal, TextInput, TouchableOpacity } from 'react-native';
-import { DataStore } from 'aws-amplify';
-import { Todo } from './models';
+import React, {useState, useEffect} from 'react';
+import {
+  StyleSheet,
+  Text,
+  View,
+  FlatList,
+  Pressable,
+  Modal,
+  TextInput,
+  TouchableOpacity,
+} from 'react-native';
+import {DataStore} from 'aws-amplify';
+import {Todo} from './models';
 
-export default function Home() {
+const Header = () => (
+  <View style={styles.headerContainer}>
+    <Text style={styles.headerTitle}>Todo App</Text>
+  </View>
+);
+
+const AddModal = ({modalVisible, setModalVisible}) => {
+  const [todoName, setTodoName] = useState('');
+  const [todoDescription, setTodoDescription] = useState('');
+
+  async function addTodo() {
+    await DataStore.save(
+      new Todo({
+        name: todoName,
+        description: todoDescription,
+        isComplete: false,
+      })
+    );
+    setModalVisible(false);
+    setTodoName('');
+    setTodoDescription('');
+  }
+
+  function closeModal() {
+    setModalVisible(false);
+  }
+
+  return (
+    <Modal
+      animationType="fade"
+      transparent={true}
```

Comment:
Nit: [use JSX shorthand](https://reactjs.org/docs/jsx-in-depth.html#props-default-to-true) for declaring `true` prop

Reply:
Thanks. This is really helpful